src/modules/auth/components/signup/signup-form/signup-form.tsx

```diff
@@ -34,7 +40,16 @@ export function SignupForm() {
   }
 
   function onSubmit(data: SignupFormData) {
-    console.log(data);
+    axiosInstance
+      .post(ENDPOINTS.signup, data)
+      .then((res) => res.data)
+      .then((res) => {
+        AccessTokenStore.setToken(res.data.accessToken);
+        navigate(ENDPOINTS.root);
+      })
+      .catch((err) => {
+        setError('email', { message: err.response.data.message });
```

Comment:
Add type checking for `err.response`; an error might have been thrown by network issues that do not have a response object.

Reply:
fixed